scenarioo-client/app/step/step.controller.js

```diff
@@ -21,7 +21,7 @@ function StepController($scope, $routeParams, $location, $route, StepResource, H
                         $filter, ApplicationInfoPopupService, GlobalHotkeysService, LabelConfigurationsResource,
                         SharePageService, SketcherContextService, RelatedIssueResource, SketchIdsResource,
                         SketcherLinkService, BranchesAndBuildsService, ScreenshotUrlService, SelectedComparison, BuildDiffInfoResource,
-                        StepDiffInfoResource, DiffInfoService, localStorageService) {
+                        StepDiffInfoResource, DiffInfoService, localStorageService, ConfigService) {
```

Comment:
Yet another dependency. Couldn't it be integrated into an existing service?

Reply:
@danielsuter haha, are you joking?? ;-) there are tooooo much dependencies already, sure. but this component needs more serious refactoring than what I had time for. I think the whole step component should be broken up into sub components. E.g. the comparison tab should be its own component (In my oppinion: should have been designed like that right from the start!). And we should also think about refactoring some of the dependencies, but I definitvely didnt have time for refactoring all that as part of the small changes I wanted to do. If you see how this dependency could have been avoided very easily by not introducing too much needless complexity, feel free to change it! To me it doesnt look reasonable to get the property from one of the other services. None of them sounds the place to be for that. And also I do not like to touch two kind of services just for passing a simple config property into the controller or view where it is needed. So all in all: good topic to discuss once, but not something to be changed fast without discussing together and thinking about what would be the best structure for the future. Maybe you want to open an issue to refactor that once?